### PR TITLE
Modify deployments per ns test to use workloads framework

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit:
- Modifies the templates, build script to call workloads framework
  and updates the properties file with the new params.
- Updates the readme with the status.

### CODE REVIEW

**Comment by akrzos** [comment]
@chaitanyaenr looks like we ended up with a merge conflict after the previous merge

**Comment by chaitanyaenr** [comment]
@akrzos rebased to resolve the conflict. PTAL. Thanks.

File: jjb/dynamic/scale-ci_deployments_per_ns.yml [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        description: The user for the Orchestration host.
+        name: ORCHESTRATION_USER
+    - string:
+        default: "quay.io/openshift-scale/scale-ci-workload"
+        description: Container image that runs the workload script.
+        name: WORKLOAD_IMAGE
+    - bool:
+        default: false
+        description: Enables/disables the node selector that places the workload job on the `pbench` node.
+        name: WORKLOAD_JOB_NODE_SELECTOR
```
Comment (akrzos) [inline jjb/dynamic/scale-ci_deployments_per_ns.yml]: These values are set to `true` for a default right now - https://github.com/openshift-scale/workloads/blob/master/workloads/vars/deployments-per-ns.yml#L17

As I have been adding pass/fail functionality I have been setting as many defaults to support run anywhere vs run with as much instrumentation as possible.  Thus that is why nodevertical/podvertical are set to false for those values

File: jjb/dynamic/scale-ci_deployments_per_ns.yml [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        description: Enables/disables running the workload wrapped by pbench-user-benchmark. When enabled, pbench agents can then be enabled (ENABLE_PBENCH_AGENTS) for further instrumentation data and pbench-copy-results can be enabled (ENABLE_PBENCH_COPY) to export captured data for further analysis.
+        name: PBENCH_INSTRUMENTATION
+    - bool:
+        default: false
+        description: Enables/disables the collection of pbench data on the pbench agent pods. These pods are deployed by the tooling playbook.
+        name: ENABLE_PBENCH_AGENTS
+    - bool:
+        default: false
+        description: Enables/disables the copying of pbench data to a remote results server for further analysis.
+        name: ENABLE_PBENCH_COPY
```
Comment (akrzos) [inline jjb/dynamic/scale-ci_deployments_per_ns.yml]: This var doesn't exist for this job.

File: jjb/dynamic/scale-ci_deployments_per_ns.yml [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+        description: Enables/disables running the workload pod as privileged.
+        name: WORKLOAD_JOB_PRIVILEGED
+    - string:
+        default: "/root/.kube/config"
+        description: Location of kubeconfig on orchestration host.
+        name: KUBECONFIG_FILE
+    - bool:
+        default: false
+        description: Enables/disables running the workload wrapped by pbench-user-benchmark. When enabled, pbench agents can then be enabled (ENABLE_PBENCH_AGENTS) for further instrumentation data and pbench-copy-results can be enabled (ENABLE_PBENCH_COPY) to export captured data for further analysis.
+        name: PBENCH_INSTRUMENTATION
```
Comment (akrzos) [inline jjb/dynamic/scale-ci_deployments_per_ns.yml]: This var doesn't exist for this job (yet).

File: jjb/dynamic/scale-ci_deployments_per_ns.yml (hunk 1 above)
Comment (chaitanyaenr) [reply jjb/dynamic/scale-ci_deployments_per_ns.yml]: Got it, we can stick with false in the templates I guess. I will create a PR to update the workload defaults.

**Comment by chaitanyaenr** [comment]
@akrzos made the changes requested, PTAL.

**Pull request merged**